packages/legend-query-builder/src/components/QueryBuilderParametersPanel.tsx

```diff
@@ -370,7 +370,7 @@ export const QueryBuilderParametersPanel = observer(
           <div className="panel__header__title">
             <div className="panel__header__title__label">parameters</div>
           </div>
-          {!isReadOnly && queryBuilderState.isParameterSupportDisabled && (
+          {!isReadOnly && !queryBuilderState.isParameterSupportDisabled && (
```

Comment:
Do we need to have this check `!isReadOnly && !queryBuilderState.isParameterSupportDisabled` again in `addParameter` function because we already have made similar checks when we are using the function

Reply:
hm, technically no, but I prefer to be defensive here.